test/e2e_source_map_test.ts

```diff
@@ -22,9 +22,6 @@ import {createOutputRetainingCompilerHost, createSourceReplacingCompilerHost, to
 
 import * as testSupport from './test_support';
 
-describe('source maps with TsickleCompilerHost', () => {
```

Comment:
Ah, ok. Yes, maybe it is worth splitting them into an own file as there are not many (before your change).